src/main/java/mate/academy/hibernate/relations/exception/DataProcessingException.java

```
@@ -0,0 +1,10 @@
+package mate.academy.hibernate.relations.exception;
+
+public class DataProcessingException extends RuntimeException {
+    public DataProcessingException(String message, Exception e) {
+        super(message, e);
+    }
+
+    public DataProcessingException(String message) {
+    }
```

This constructor is missing a call to `super(message)`. Without it, the exception message provided when creating the exception will be lost, and `e.getMessage()` will return `null`.